pkg/controller/pkg/revision/deployment.go

```diff
@@ -34,7 +34,7 @@ var (
 	runAsNonRoot             = true
 )
 
-func buildProviderDeployment(provider *metav1alpha1.Provider, revision v1alpha1.PackageRevision, namespace string) (*corev1.ServiceAccount, *appsv1.Deployment) { // nolint:interfacer
+func buildProviderDeployment(provider *metav1alpha1.Provider, revision v1alpha1.PackageRevision, controllerConfig *v1alpha1.ControllerConfig, namespace string) (*corev1.ServiceAccount, *appsv1.Deployment) { // nolint:interfacer,gocyclo
```

Comment:
Does we benefit from this lengthy, explicit variable name given its type is `ControllerConfig`? Would `cc *v1alpha1.ControllerConfig` suffice? 😬